docs/ML_DGA.md

```diff
@@ -0,0 +1,175 @@
+# Machine Learning on Domain Generation Algorithm (DGA)
```

Comment:
should we link to this document from anywhere else?

Reply:
yes, I will add a link to it in every related release as a reference

Reply:
oh i mean from other markdown files, like README.md